frontend/src/metabase/core/components/AccordionList/AccordionList.jsx

```diff
@@ -124,6 +123,7 @@ export default class AccordionList extends Component {
     getItemClassName: item => item.className,
     getItemStyles: item => {},
     hasInitialFocus: true,
+    showSpinner: item => false,
```

Comment:
if you want to show the passed value is `item` but don't want to use it you can append `_` in front of its name. I don't why this isn't caught by ESLint, but let's make sure all unused parameter have `_` prefix.

```suggestion
    showSpinner: _item => false,
```